main.py

```diff
@@ -1,10 +1,68 @@
 import init_django_orm  # noqa: F401
 
+import json
+from datetime import datetime
+
 from db.models import Race, Skill, Player, Guild
 
 
 def main() -> None:
-    pass
+    with open("players.json", "r") as file:
+        players_data = json.load(file)
+
+    for player_nickname, player in players_data.items():
+
+        # Creating a race
+
+        race_name = player.get("race").get("name")
+        race_description = player.get("race").get("description")
```

Comment:
First get is not needed use brackets or do as follow:
player.get("race", dict()).get("name")
Fin in other places